modules/deployment-model-impl/src/main/java/org/mule/runtime/module/deployment/impl/internal/policy/DefaultApplicationPolicyInstance.java

```diff
@@ -110,7 +111,8 @@ private void initPolicyContext() throws InitialisationException {
       customizationService.overrideDefaultServiceImpl(OBJECT_POLICY_MANAGER_STATE_HANDLER,
```

Comment:
Add a comment explaining why this object is not proxied

Reply:
Done.